include/rmm/detail/dynamic_load_runtime.hpp

```diff
@@ -27,25 +27,23 @@ namespace rmm::detail {
  * are added in newer minor versions of the cuda runtime.
  */
 struct dynamic_load_runtime {
-  static constexpr auto dlclose_destructor = [](void* handle) { ::dlclose(handle); };
-  inline static std::unique_ptr<void, decltype(dlclose_destructor)> cuda_runtime_lib{
-    nullptr, dlclose_destructor};
-
-  static bool open_cuda_runtime()
+  static void* open_cuda_runtime()
```

Comment:
I'd just call it `get_cuda_runtime_handle` now since it only opens on the first invocation. 
```suggestion
  static void* get_cuda_runtime_handle()
```

Reply:
Done